ui-components/src/headers/SiteHeader.tsx

```diff
@@ -113,7 +113,7 @@ const SiteHeader = (props: SiteHeaderProps) => {
 
     return options.map((option, index) => {
       return (
-        <div key={index}>
+        <div className="navbar-dropdown-item-container" key={index}>
```

Comment:
Similarly, is this a change that upstream needs? If not, maybe make this more general by taking in a className prop and then have us set our own class in public/

Reply:
I think it's OK to hardcode this className here, as it's done similarly elsewhere in this file (`navbar-dropdown-*`).

Reply:
Ok, we will just need to upstream this change as well.

Reply:
I split this change out into a separate commit to make upstreaming easer.